doc/src/content/atoms/dropdown/props.md

```diff
@@ -0,0 +1,19 @@
+---
+path: "/dropdown/props"
+component: "DropDown"
+type: "atom"
+title: "DropDown"
+---
+
+Use `DropDown` component to create a select field.
+
+| Prop | Description | Type | Default |
+| ------ | ----------- | ---- | ------- |
+| options | options to be shown for the select field | array of objects; each object should contain label and value | [] |
+| icon | component to render icon based on user's action | React.Component | Component which renders toggleable `caret` icon |
```

Comment:
React.Component is still an object ? Is it intended ?

Reply:
I meant that a component should be a prop. 
I could have written a better desc.